README.md

```diff
@@ -1,2 +1,494 @@
+
+## Running Your Tests
+
+### Run a specific test on your own machine
+
+- How to run the test?
+
+  To run the default test scenario (e.g. End to End Scenario) on your own machine, use the following command:
+
+  Maven:
+    ```sh
+  mvn test -P on-prem
+  ```
+
+  Gradle:
+    ```sh 
+  gradle on-prem
+  ```
+
+  To run a specific test scenario, use the following command with the additional 'test-name' argument:
+
+  Maven:
+  ```sh
+  mvn test -P on-prem -Dtest-name="<Test scenario name>"
+  ```
+
+  Gradle:
+  ```sh
+  gradle on-prem -Dtest-name="<Test scenario name>"
+  ```
+
+  where,  the argument 'test-name' can be any Cucumber scenario name configured in this repository.
+
+  E.g. "Login as username", "Login as Locked User", "Offers for mumbai geo-location" or any of the other test scenario names, as outlined in [About the tests in this repository](#About-the-tests-in-this-repository) section.
+
+- Output
+
+  This run profile executes a specific test scenario on a single browser instance on your own machine.
+
+
+### Run the entire test suite on your own machine
+
+- How to run the test?
+
+  To run the entire test suite on your own machine, use the following command:
+
+  Maven:
+  ```sh
+  mvn test -P on-prem-suite
+  ```
+
+  Gradle:
+  ```sh
+  gradle on-prem-suite
+  ```
+
+- Output
+
+  This run profile executes the entire test suite sequentially on a single browser, on your own machine.
+
+  
+---
+
+# Docker
+
+[Docker](https://docs.docker.com/get-started/overview/) is an open source platform that provides the ability to package and test applications in an isolated environment called containers.
+
+## Prerequisites
+
+- Install and start [Docker](https://docs.docker.com/get-docker/).
+- Note: Docker should be running on the test machine. Ensure Docker Compose is installed as well.
+- Run `docker-compose pull` from the current directory of the repository.
+
+## Running Your Tests
+
+### Run a specific test on the docker infrastructure
+
+- How to run the test?
+
+  - Start the Docker by running the following command:
+
+  ```sh
+  docker-compose up -d
+  ```
+
+  - To run the default test scenario (e.g. End to End Scenario) on your own machine, use the following command:
+
+  Maven:
+  ```sh
+  mvn test -P docker
+  ```
+
+  Gradle:
+    ```sh
+  gradle docker
+  ```
+
+  To run a specific test scenario, use the following command with the additional 'test-name' argument:
+
+  Maven:
+  ```sh
+  mvn test -P docker -Dtest-name="<Test scenario name>"
+  ```
+
+  Gradle:
+  ```sh
+  gradle docker -Dtest-name="<Test scenario name>"
+  ```
+
+  where,  the argument 'test-name' can be any Cucumber scenario name configured in this repository.
+
+  E.g. "Login as username", "Login as Locked User", "Offers for mumbai geo-location" or any of the other test scenario names, as outlined in [About the tests in this repository](#About-the-tests-in-this-repository) section.
```

Comment:
`mvn test -P on-prem -Dtest-name="Offers for mumbai geo-location"`
`mvn test -P on-prem -Dtest-name="Login as username"`

Both these profiles do not trigger any test.

Reply:
The profile names are different than the one's mentioned in the README.md
This is the right command:
 `mvn test -P on-prem -Dtest-name="Login with given '<username>' user" `
` mvn test -P on-prem -Dtest-name="Offers for Mumbai location"`        
We will make the changes.